stream-testkit/src/test/scala/org/apache/pekko/stream/testkit/StreamSpec.scala

```diff
@@ -27,10 +26,26 @@ import pekko.testkit.{ PekkoSpec, TestProbe }
 import pekko.testkit.TestKitUtils
 
 import org.scalatest.Failed
+import org.scalatest.time.{ Millis, Span }
 
 import com.typesafe.config.{ Config, ConfigFactory }
 
+import java.util.concurrent.TimeUnit
+
 abstract class StreamSpec(_system: ActorSystem) extends PekkoSpec(_system) {
+
+  final case class StreamConfig(allStagesStoppedTimeout: Span = Span({
```

Comment:
This might look a bit un-orthodox by Pekko standards, but it is the idiomatic way to override a configuration when in ScalaTest (which is what `StreamSpec` extends).

The defaault behaviour is still the same (i.e. getting the value from `pekko.stream.testkit.all-stages-stopped-timeout`) but if you decide to override it then you can do it the ScalaTest specific way.